src/cmds/openapi/uses.ts

```diff
@@ -0,0 +1,282 @@
+          hasUnusedFeature = true;
+        } else {
+          report.push('');
+          report.push(`${feature}:`);
+          report.push(...(info.locations as string[]).map(loc => ` · ${chalk.yellow(loc)}`));
+        }
+      }
+    });
+
+    if (features.includes('readme')) {
+      // Add some spacing between our OpenAPI and ReadMe extension reports (but only if our last
+      // entry wasn't an empty line).
+      if (features.length > 1 && report[report.length - 1].length) {
+        report.push('');
+      }
+
+      Object.entries(analysis.readme).forEach(([feature, info]) => {
+        if (!info.present) {
+          report.push(`${feature}: You do not use this.`);
+          hasUnusedFeature = true;
+        } else {
+          report.push(`${feature}:`);
+          report.push(...(info.locations as string[]).map(loc => ` · ${chalk.yellow(loc)}`));
+          report.push('');
+        }
+      });
+    }
+
+    // Because we add a little bit of padding between our report and the "analyzing your spec" copy
+    // if this second entry in the report is an empty line then we can safely remove it so we don't
+    // end up with multiple empty lines at the top of our report.
+    if (!report[1].length) {
+      report.splice(0, 1);
+    }
+
+    // If the last entry in our report array is an empty string then we should remove it.
+    if (!report[report.length - 1].length) {
+      report.pop();
+    }
+
+    return {
+      report: report.join('\n'),
+      hasUnusedFeature,
+    };
+  }
+
+  buildFullReport(analysis: Analysis) {
+    const report: string[] = ['Here are some interesting things we found in your API defintion. 🕵️', ''];
+
+    // General API definition statistics
+    Object.entries(analysis.general).forEach(([, info]) => {
+      let msg: string;
+
+      if (Array.isArray(info.found)) {
+        if (!info.found.length) {
+          return;
+        }
+
+        const highlightedData = info.found.map(d => chalk.yellow(d));
+        if (info.found.length > 1) {
+          msg = `You are using ${chalk.bold(info.found.length)} ${pluralize(
+            info.name,
+            info.found.length
+          )} throughout your API: ${new Intl.ListFormat('en').format(highlightedData)}`;
+        } else {
+          msg = `You are using a single ${info.name} throughout your API: ${highlightedData[0]}`;
+        }
+      } else if (info.found > 1) {
+        msg = `You have a total of ${chalk.bold(info.found)} ${pluralize(info.name, info.found)} in your API.`;
+        if (info.found > 100) {
+          msg += ' Wow!';
+        }
+      } else {
+        msg = `You have a single ${info.name} in your API.`;
+      }
+
+      report.push(` · ${msg}`);
+    });
+
+    // Build out a view of all OpenAPI and ReadMe features that we discovered.
+    [
+      { component: 'openapi', header: 'OpenAPI Features' },
+      { component: 'readme', header: 'ReadMe-specific Features and Extensions' },
```

Comment:
Per AP-style title casing:
```suggestion
      { component: 'readme', header: 'ReadMe-Specific Features and Extensions' },
```

Reply:
![giphy](https://user-images.githubusercontent.com/33762/206582062-78d24ab2-904c-4442-9ef2-b61e43ccdb46.gif)